src/components/nodes/sequenceFlow/index.js

```diff
@@ -65,7 +65,7 @@ export default {
           items: [
             {
               component: 'FormInput',
-              config: nameConfigSettings,
+              config: { ...nameConfigSettings, validation: null },
```

Comment:
Should these still be required fields? The Issue did not specify.

Reply:
I just confirmed with Alan and nope, we should leave it optional; which makes sense from a UX perspective; less friction for the user as they aren't given an invalid element by default.